kokoro/ubuntu/continuous.cfg

```diff
@@ -0,0 +1,4 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Location of the continuous bash script in repository.
+build_file: "appengine-plugins-core/kokoro/ubuntu/continuous.sh"
```

Comment:
We'll create separate directories for Win and Mac under the `kokoro` directory and they'll refer their respective files.